ember_debug/libs/octane-tree.js

```diff
@@ -0,0 +1,14 @@
+export function makeRenderNodeCloneable(retainObject, renderNode) {
```

Comment:
nit: the file _maybe_ misnamed, since it would work on 3.14+ "non-Octane" apps too, not sure if it matters

Reply:
Fair enough. I was just thinking of a file like the current `glimmer-tree` one. What would be a good name for it? I suppose we could put it all in the existing `glimmer-tree` and do the version checks there.

Reply:
It may be a fine name as long as we don’t be tempted into thinking we can do anything octane specific (such as assuming the template-only semantics)